docs/topics/proc_attaching-a-remote-debugger-to-the-application.adoc

```diff
@@ -0,0 +1,33 @@
+
+[#attaching-a-remote-debugger-to-the-application]
+= Attaching a Remote Debugger to the Application
+
+When your application is configured for debugging, attach a remote debugger of your choice to it.
+In this guide, The xref:https://www.redhat.com/en/technologies/jboss-middleware/developer-studio[Red Hat JBoss Developer Studio] is covered, but the procedure is similar when using other programs.
+
+.Prerequisites
+
+* The application running either locally or on OpenShift, and configured for debugging.
+* The port number that your application is listening on for debugging.
+* Red Hat JBoss Developer Studio installed on your machine. You can download it from the xref:https://developers.redhat.com/products/devstudio/download/[Red Hat JBoss Developer Studio download page].
+
+.Procedure
+
+. Start Red Hat JBoss Developer Studio.
+. Create a new debug configuration for your application:
+.. Click *Run->Debug Configurations*.
+.. In the list of configurations, double-click *Remote Java application.*
+This creates a new remote debugging configuration.
+.. Enter a suitable name for the configuration in the *Name* field.
+.. Enter the path to the directory with your application into the *Project* field. You can use the *Browse...* button for convenience.
+.. Set the *Connection Type* field to _Standard (Socket Attach)_ if it is not already.
+.. Set the *Port* field to the port number that your application is listening on.
```

Comment:
Should probably highlight that it's the _debugger port_, not the common application port.